internal/log/logger.go

```diff
@@ -0,0 +1,32 @@
+package log
+
+import (
+	"os"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/rs/zerolog/pkgerrors"
+)
+
+func InitLogger() {
+	// overrides zerolog global logger
+	log.Logger = NewLogger("default")
+}
+
+func NewLogger(name string) zerolog.Logger {
+	zerolog.ErrorStackMarshaler = pkgerrors.MarshalStack
+
+	level := zerolog.WarnLevel
+	if lvl, err := zerolog.ParseLevel(os.Getenv("LOG_LEVEL")); err == nil && lvl != zerolog.NoLevel {
+		level = lvl
+	}
+	zerolog.SetGlobalLevel(level)
+
+	prettify := os.Getenv("LOG_PRETTIFY") == "true"
```

Comment:
shouldn't this rely on a more general env var to determine prod or dev env instead of a logging specific var?

Reply:
I thought about it, but decided to go for a specific env var, because it gives more control.
I don't think hiding many functionalities (in the future) behind a single flag is a good idea